java/test/jmri/jmrit/roster/swing/speedprofile/SpeedProfilePanelTest.java

```diff
@@ -14,6 +16,7 @@
 
     @Test
     public void testCTor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
As stated before, we don't generally need or want the graphics check on panels, especially in constructor tests.

Reply:
Done!

Reply:
That change is still showing as it did before.  Did you remember to commit the file?

Reply:
Looks correct here... and not showing as changed in GitHub Desktop… Maybe it committed when the other one I'd missed got (re-?)committed.